wee_slack.py

```diff
@@ -2400,6 +2401,11 @@ def resolve_ref(ref):
         # Something else, just return as-is
     return ref
 
+def create_user_status_string(profile):
+    status_emoji = profile.get("status_emoji") if profile.get("status_emoji") else "None"
+    status_text = profile.get("status_text") if profile.get("status_text") else "None"
+    return "[{}] {}".format(status_emoji, status_text)
```

Comment:
Yep, that currently means that the status will show as `[None] None`. 

It was showing just `None` before my proposed changes, and I liked that more than having the topic empty when I tried it - An empty topic made me wonder whether the user had no status at all, or if setting the topic simply wasn't working.

So that said, my personal preference would be to at least indicate that there's no status _somehow_, even if `[None] None` isn't the best way. Any suggestions? Perhaps just stating it as "_username_ has no status", something like that?